telescope/utils.py

```diff
@@ -324,7 +337,7 @@ async def fetch(self, project: str, name: str) -> List[BugInfo]:
                     buglist = await fetch_json(
                         url, headers={"X-BUGZILLA-API-KEY": config.BUGTRACKER_API_KEY}
                     )
```

Comment:
You may want to add an assertion here like `assert "bugs" in buglist` ?

Reply:
```Use of assert detected. The enclosed code will be removed when compiling to optimised byte code```
That seems to not be useful or am I misunderstanding the lint error?

Reply:
Yeah, when your run the app with `python -O`, assertions are not executed, which can be misleading

Reply:
We can ignore the rule I think since we don't run with `-O` or `PYTHONOPTIMIZE=1`

Reply:
Ended up adjusting to handle it another way. What do you think?